lib/RNCryptor.js

```diff
@@ -1,12 +1,11 @@
 (function() {
   var crypto = require('crypto');
-  var MCrypt = require('mcrypt').MCrypt;
 
   var _settings = {};
 
   var _configure_settings = function(version) {
     var settings = {
-      algorithm: 'rijndael-128',
+      algorithm: 'aes-256-cbc',
```

Comment:
are `rijndael-128` and `aes-256-cbc` compatible would these break implementations that need backward compatibility?